library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -220,15 +225,29 @@ spdm_process_opaque_data_supported_version_data(IN spdm_context_t *spdm_context,
          SECURED_MESSAGE_OPAQUE_ELEMENT_SMDATA_DATA_VERSION) ||
         (opaque_element_support_version->sm_data_id !=
          SECURED_MESSAGE_OPAQUE_ELEMENT_SMDATA_ID_SUPPORTED_VERSION) ||
-        (opaque_element_support_version->version_count != 1)) {
+        (opaque_element_support_version->version_count < 0)) {
```

Comment:
How could unsigned value < 0 ???

I think we need use "opaque_element_support_version->version_count == 0"